K2Bridge/ElasticQueryTranslator.cs

```diff
@@ -104,6 +104,13 @@ public QueryData TranslateQuery(string header, string query)
                 docValueFields = new List<string>();
                 elasticSearchDsl.DocValueFields?.ForEach(item => docValueFields.Add(item.Field));
 
+                // We currently only support a single primary aggregation, so we take the first element from the dictionary and retrieve its type name
```

Comment:
can there be 2 top level aggs?

Reply:
In theory, I think yes, i.e. Elastic supports it, but so far I have not seen a case where Kibana needs it...